spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+describe Api::V1::UsersController do
+  before(:each) { request.headers['Accept'] = "application/vnd.birthday_finder_api.v1" }
```

Comment:
Line is too long. [88/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.